org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/hover/FocusableBrowserInformationControl.java

```diff
@@ -137,7 +137,7 @@ private static Object safeEvaluate(Browser browser, String expression) {
 	private static boolean safeExecute(Browser browser, String expression) {
 		try {
 			return browser.execute(expression);
-		} catch (Throwable ex) {
+		} catch (Exception ex) {
```

Comment:
I know some tools discourage catching Throwable, but at this point, I think it is what we want. No matter what it happens, we want to log it and continue as normal.